src/proxy.c

```diff
@@ -1606,7 +1606,9 @@ cc_proxy_hyper_exec_command (struct cc_oci_config *config)
 		json_array_add_object_element (envs, env_var);
 	}
 
-	json_object_set_string_member (process_node, "workdir", process->cwd);
+	if (process->cwd[0]) {
```

Comment:
This should segfault if cwd is null. That said the rest of the code is also doing this check without checking if cwd if null beforehand, so probably missing something.